ddtrace/appsec/utils.py

```diff
@@ -1,25 +1,92 @@
+import base64
 import os
+import sys
+from typing import TYPE_CHECKING
 
+from ddtrace import constants
 from ddtrace.constants import APPSEC_ENV
 from ddtrace.internal.utils.formats import asbool
 
 
+if TYPE_CHECKING:  # pragma: no cover
+    from typing import Text
+
+
 def _appsec_rc_features_is_enabled():
     # type: () -> bool
     if asbool(os.environ.get("DD_REMOTE_CONFIGURATION_ENABLED", "true")):
         return APPSEC_ENV not in os.environ
     return False
 
 
+def to_bytes_py2(n, length=1, byteorder="big", signed=False):
```

Comment:
nit: Maybe move this to `ddtrace.internal.compat` and add some comments on what does it do?

Reply:
Yup. Good call.